packages/components/ovh-at-internet/src/config.ts

```diff
@@ -1,15 +1,63 @@
 import { isEmpty } from 'lodash-es';
+import { CountryCode } from '@ovh-ux/manager-config/types/locale';
+
+// eslint-disable-next-line prettier/prettier
```

Comment:
Seen together, we'll schedule a bump of both ESLint and Prettier beginning of next year.
Relates to:
- #4630